### ISSUE TITLE
ComboBox selection fails on REPL

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#79]
**Expected behavior**
Valid commands should not fail on the REPL

**Actual behavior**
Getting an error
```
TypeError: comboBox(...).select is not a function
    at repl:2:41
    at repl:3:3
    at Script.runInContext (vm.js:102:20)
    at Object.defaultEval (repl.js:319:29)
    at bound (domain.js:396:14)
    at Object.runBound (domain.js:409:12)
```
**Steps to replicate**
* Open the REPL
* openBrowser()
* goto("google.com")
* click("I'm Feeling Lucky")
* comboBox({"id":"lang-chooser"}).select("‪हिन्दी‬")
> Note the error is displayed. I am able to run this from the recorded script

**Version**
```
commit - e72d723ce592aa92ffef44c9d371ea01dbf1acca
Date - Tue Jul 24 11:37:30 2018 +0530
```

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 21c9289

**Comment by sguptatw** [comment]
**Verified with version**
```
commit 215a0767e673443e8f91ddaa0a56e984bad09a94
```
